### DISCUSSION TITLE
Saving Split Tabs

### DISCUSSION THREAD

**Discussion by thesoloist-1** [zen-browser/desktop#1175] [Ideas]
Currently when you use split tabs and close the Zen browser, the split tabs are not restored when reopening the browser. It would be great if there was an option to save those split tabs so when you reopen the browser, the split tabs are still there.

**Comment by realSilasYang** [comment]
Great proposal

**Comment by TW527E** [comment]
+1 I thought it is a great feature of split tabs 👏

**Comment by cjil** [comment]
I only just recently (accidentally) discovered the split tabs.

The only issue I have with them is that the state is lost when closing Zen. This proposal would take this feature from being good, to amazing..

+1 for saving the split tabs
